### PR TITLE
Classic App Insights to LA - Solutions Packaging

### PR DESCRIPTION
**Pull request by v-amolpatil**
Required items, please complete
   
   Change(s):
   - Classic app insights to Log analytic workspace changes for below listed solutions.
   - Solutions for which packaging is done are: AWS Systems Manager, AWSAthena, DomainTools, Fortinet FortiGate Next-Generation Firewall connector for Microsoft Sentinel.
   - Solutions for which packaging is not needed are: Neustar IP GeoPoint, AWS_IAM as changes are in azuredeploy and no need to repackage.
   - For "DomainTools" and "Fortinet FortiGate Next-Generation Firewall connector for Microsoft Sentinel" solution there are additional changes because there were either arm-ttk failures, incorrect mapping to file path i.e. link or failing while deployment so you will see additional file changes as we have fixed this.

   Reason for Change(s):
   - Stated Above

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

Testing documents:
[DomainTools Solution.docx](https://github.com/Azure/Azure-Sentinel/files/14095609/DomainTools.Solution.docx)
[Fortinet Fortigate Next Generation Firewall Solution.docx](https://github.com/Azure/Azure-Sentinel/files/14095595/Fortinet.Fortigate.Next.Generation.Firewall.Solution.docx)
[AWS System Manager Solution.docx](https://github.com/Azure/Azure-Sentinel/files/14095622/AWS.System.Manager.Solution.docx)
[AWSAthena Solution.docx](https://github.com/Azure/Azure-Sentinel/files/14095623/AWSAthena.Solution.docx)

### CODE REVIEW

File: Solutions/AWS Systems Manager/Package/mainTemplate.json [hunk 1]
```diff
@@ -60,47 +60,47 @@
     "playbookContentId3": "AWS-SSM-GetInstancePatches-HostEntityTrigger",
     "_playbookContentId3": "[variables('playbookContentId3')]",
     "playbookId3": "[resourceId('Microsoft.Logic/workflows', variables('playbookContentId3'))]",
-    "playbookTemplateSpecName3": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId3'))),variables('playbookVersion3')))]",
+    "playbookTemplateSpecName3": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId3'))))]",
```
Comment (daxesht) [inline Solutions/AWS Systems Manager/Package/mainTemplate.json:63]: why this has been removed?

File: Solutions/AWS Systems Manager/Package/mainTemplate.json [hunk 2]
```diff
@@ -60,47 +60,47 @@
...
     "_playbookcontentProductId3": "[concat(take(variables('_solutionId'),50),'-','pl','-', uniqueString(concat(variables('_solutionId'),'-','Playbook','-',variables('_playbookContentId3'),'-', variables('playbookVersion3'))))]",
     "AWS-SSM-GetInstancePatches-IPEntityTrigger": "AWS-SSM-GetInstancePatches-IPEntityTrigger",
     "_AWS-SSM-GetInstancePatches-IPEntityTrigger": "[variables('AWS-SSM-GetInstancePatches-IPEntityTrigger')]",
     "playbookVersion4": "1.0",
     "playbookContentId4": "AWS-SSM-GetInstancePatches-IPEntityTrigger",
     "_playbookContentId4": "[variables('playbookContentId4')]",
     "playbookId4": "[resourceId('Microsoft.Logic/workflows', variables('playbookContentId4'))]",
-    "playbookTemplateSpecName4": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId4'))),variables('playbookVersion4')))]",
+    "playbookTemplateSpecName4": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId4'))))]",
```
Comment (daxesht) [inline Solutions/AWS Systems Manager/Package/mainTemplate.json:71]: why this has been removed?

**Review by daxesht — COMMENTED** [review]
Please check whole PR changes, It should only contains changes related to classic app insight deprecation.

File: Solutions/AWS Systems Manager/Package/mainTemplate.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
     "playbookContentId2": "AWS-SSM-GetInstancePatches",
     "_playbookContentId2": "[variables('playbookContentId2')]",
     "playbookId2": "[resourceId('Microsoft.Logic/workflows', variables('playbookContentId2'))]",
-    "playbookTemplateSpecName2": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId2'))),variables('playbookVersion2')))]",
+    "playbookTemplateSpecName2": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat(parameters('workspace'),'-pl-',uniquestring(variables('_playbookContentId2'))))]",
```
[the remark this answers is not in the document]
Comment (v-amolpatil) [reply Solutions/AWS Systems Manager/Package/mainTemplate.json:54]: There was a tool change for which we removed the version 5months back. Earlier package 3.0.0 was not having this change so now after we package with new 3.0.1 we are seeing this change. This change in tool is done for all resource types.

File: Solutions/AWS Systems Manager/Package/mainTemplate.json (hunk 1 above)
Comment (v-amolpatil) [reply Solutions/AWS Systems Manager/Package/mainTemplate.json:63]: explained in previous comment

File: Solutions/AWS Systems Manager/Package/mainTemplate.json (hunk 2 above)
Comment (v-amolpatil) [reply Solutions/AWS Systems Manager/Package/mainTemplate.json:71]: explained in previous comment

**Review by daxesht — DISMISSED** [review]

**Review by daxesht — APPROVED** [review]

**Pull request merged**